src/ol_infrastructure/applications/xpro/Pulumi.applications.xpro.QA.yaml

```diff
@@ -25,6 +25,7 @@ config:
     MITXPRO_LOG_LEVEL: "INFO"
     MITXPRO_SECURE_SSL_HOST: "rc.xpro.mit.edu"
     MITXPRO_USE_S3: "True"
+    MITXPRO_SUPPORT_EMAIL: "support@xpro-rc-mail.odl.mit.edu"
```

Comment:
@pdpinch , could you please confirm if the email support@xpro-rc-mail.odl.mit.edu is appropriate for our use in the RC environment? however, for prod, it is support@xpro.mit.edu.